packages/ui/origin-ui-v2/libs/ui/shared-state/src/token/localStorage.ts

```diff
@@ -0,0 +1,18 @@
+import { AUTHENTICATION_TOKEN } from '../utils';
+import {
+  TGetAuthenticationToken,
+  TRemoveAutheticationToken,
+  TSetAuthenticationToken,
+} from './types';
+
+export const setAuthenticationToken: TSetAuthenticationToken = (token) => {
```

Comment:
We probably want additional param ( smth like `noReplace` which should probably be true by default ).

Reply:
resolution -> not necessary.